Wise_Game/Wise_Game/main.swift

```diff
@@ -0,0 +1,344 @@
+//
+//  main.swift
+//  TextAdventure
+//
+//  Created by Amy Alsaydi on 10/1/19.
+//  Copyright © 2019 Amy Alsaydi. All rights reserved.
+//
+
+/*
+ Assignment Requirements:
+ - no compiler errors/warnings ✅
+ - 10 user decisions✅
+ - user enter number ✅
+ - switch statement ✅
+ - if/else statment ✅
+ - type annotation ✅
+ - type inference ✅
+ - 1 tuple ✅
+ - 1 range ✅
+ - for loop ✅
+ - While loop ✅
+ 
+ */
+
+import Foundation
+
+print("WAKE UP YOU GOT US INTO THIS MESS!!!!")
+sleep(3)
+print("Some one just slid a note ✉️ under the door, should I open it?")
+let answer = readLine()?.lowercased()
+print()
+if answer == "yes" {
+    print("Uh.. Ok.. ")
+} else {
+    print("I have no idea why Im even asking you")
+}
+print()
+sleep(1)
+print("It says........check under the painting... May the odds be ever in your favor.... wait oops that may be a copyright issue. Nevermind but good luck this should be fun to watch! 🥳")
+print(); sleep(9)
+print("Theres a vault under here."); sleep(3)
+print("It says Im thinking of a number between 1 and 20, you have 4 tries.")
+sleep(3)
+let maxTries: Int = 3
+var numOfTries: Int = 0
+let codeNum = 12
+var passed = false
+
+print()
+print("Enter your choice. ")
+
+repeat {
+    if numOfTries == maxTries {
+        sleep(3)
+        print()
+        print("ACTUALLY! I lied you only get 3! 😬")
+        break
+    }
+    let choice = Int(readLine() ?? "-1") ?? -1 // readline is a string
```

Comment:
Great job for providing a default for your readLine and then a default for changing a String number to an Int